cli/runner.go

```diff
@@ -0,0 +1,305 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/exec"
+	"runtime"
+	"strings"
+)
+
+const ContainerNetName = "dyocli"
+
+func RunContainers(containers string, start bool, quiet bool) error {
+
+	//search for podman/docker
+	err, executable := findExec(true)
+	if err != nil {
+		return err
+	}
+
+	var cmd *exec.Cmd
+	if start {
+		cmd = exec.Command(executable, "up", "-d")
+	} else {
+		cmd = exec.Command(executable, "down")
+	}
+
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
+
+	log.Println("Executing", executable, "in the background...")
+	err = cmd.Run()
+	if err != nil {
+		return err
+	}
+
+	// if we want "silent" output we exit early
+	if quiet {
+		return nil
+	}
+
+	// otherwise see logs
+	log.Println("Printing logs:")
+	cmd = exec.Command(executable, "logs", "-f")
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
+
+	err = cmd.Run()
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// Find a compose executable
+func findExec(compose bool) (error, string) {
```

Comment:
`error should be returned as the last argument ST1008`

Reply:
fixed